docs/release-notes/4.tucker/4.2.0.md

```diff
@@ -33,6 +33,25 @@ HarperDB also includes support for Server-Sent Events. This is a very easy-to-us
 ## Clone Node
 HarperDB includes new functionality for adding new HarperDB nodes in a cluster. New instances can be configured to clone from a leader node, performing and copying a database snapshot from a leader node, and self-configuring from the leader node as well, to facilitate accelerated deployment of new nodes for fast horizontal scaling to meet demand needs. [See the documentation on Clone Node for more information.](../../administration/cloning.md)
 
+## Configuration
+`harperdb-config.yaml` has had some configuration values added, removed, renamed and defaults changed.
+* `operationsApi.foreground` was removed. To start HarperDB in the foreground, from the CLI run `harperdb`.
+* `clustering.republishMessages` now defaults to `false`.
+* Made `operationsApi` configuration optional. Any config not defined here will default to the `http` section.
+* Added a `securePort` parameter used for setting the https port.
+* Added a new top level `tls` section.
+* Broke out the `customFunction` configuration into the `http` and `tls` sections.
+* Removed `customFunctions.enabled`, `customFunctions.network.https`, `operationsApi.network.https` and `operationsApi.nodeEnv`
+* Added a element called `componentRoot` which replaces `customFunctions.root`.
+* Updated custom pathing to use `databases` instead of `schemas`.
+
+## Alpha Status (Current Progress)
+This is alpha release software; it is still in development with active testing. Most of the functionality in this release is working in its primary use cases, but there are many edge cases to test and refinements to finish. Here are a list of the major development works in progress:
```

Comment:
I don't know if you intended to add this section back in? (I think I had removed the alpha section)

Reply:
sorry, bad merge. Have removed.